README.md

```diff
@@ -29,11 +29,17 @@ Run `mvn install` to run unit tests, build and install the package.
 ## Run Package Locally
 To check whether the build was successful, you can start the resulting JAR file from the build process by running `java -jar target/semantic-hub-backend-{current-version}.jar`.
 
+## Precondition
```

Comment:
It would be better to add this section before the Build Packages section. 
But it is up to you :)

Reply:
incorporated the review comments.